contributing.md

```diff
@@ -55,9 +63,9 @@ git push origin features/123-boolean-operators
 
 * Enter your GitHub username and password (advanced users can remove this step by connecting to GitHub with SSH.
 
-* Go to GitHub. The new branch will show up with a green Pull Request button. Make sure the title and message are clear, concise, and self- explanatory. Then click the button to submit it.
+* Go to GitHub. The new branch will show up with a green Pull Request button. Make sure the title and message are clear, concise, and self-explanatory. Then click the button to submit it.
```

Comment:
Perhaps a statement on rebasing here?

Reply:
Hmm. Thanks for reminding me (to rebase).

Do we know what we want well enough to write it down in the guidelines? Do we want to ask the contributors to tidy up their commits only keeping the most important ones (`git rebase -i master` and then, in most cases, force push), or do we want to rebase by default before every PR? We haven't been sticking to any particular rule so far...

Reply:
They should rebase before the PR. After the PR is live, no more rebasing. Moreover, should we make a general reference with link towards our development process (what is it now btw Github Flow or Gitlab Flow?)

Reply:
I don't think we have reached an agreement on this. I would leave it all out for now.

Reply:
I've mentioned rebasing before pushing as we discussed offline.